cit3.0-web/src/utils/PrivateRoute/PrivateRoute.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Proptypes from "prop-types";
+import { Route, Redirect } from "react-router-dom";
+/**
+ * @todo: Find our sets of action interfaces
+ */
+import { useKeycloakWrapper } from "../../hooks/useKeycloakWrapper";
+
+/**
+ * A PrivateRoute only allows a user who is authenticated and has the appropriate role(s) or claim(s).
+ * @param props - Properties to pass { component, role, claim }
+ */
+const PrivateRoute = (props) => {
+  const keycloak = useKeycloakWrapper();
+  const { component: Component, layout: Layout, ...rest } = props;
+  /* eslint consistent-return: "off" */
```

Comment:
What's the issue that's causing this warning to show up?

Reply:
No default retun, I need to have more info on roles and permission before I can finish this file. I will just ad a todo for later.